modules/Bio/EnsEMBL/Analysis/Hive/Config/TranscriptSelection.pm

```diff
@@ -0,0 +1,910 @@
+    'projection_source_db_port'         => '4240',
+    'projection_source_production_name' => '',
+
+    'cdna_db_server' => $self->o('databases_server'),
+    'cdna_db_port'   => $self->o('databases_port'),
+
+    'genblast_db_server' => $self->o('databases_server'),
+    'genblast_db_port'   => $self->o('databases_port'),
+
+    'projection_db_server' => $self->o('databases_server'),
+    'projection_db_port'   => $self->o('databases_port'),
+
+    'long_read_final_db_server' => $self->o('databases_server'),
+    'long_read_final_db_port'   => $self->o('databases_port'),
+
+    'rnaseq_for_layer_db_server' => $self->o('databases_server'),
+    'rnaseq_for_layer_db_port'   => $self->o('databases_port'),
+
+    # Layering is one of the most intesnive steps, so separating it off the main output server helps
+    # Have also set module to use flatfile seq retrieval, so even if it's on the same server as the
+    # core, the core should not be accessed
+    'layering_db_server' => $self->o('dna_db_server'),
+    'layering_db_port'   => $self->o('dna_db_port'),
+
+    'utr_db_server' => $self->o('databases_server'),
+    'utr_db_port'   => $self->o('databases_port'),
+
+    'genebuilder_db_server' => $self->o('databases_server'),
+    'genebuilder_db_port'   => $self->o('databases_port'),
+
+    'pseudogene_db_server' => $self->o('databases_server'),
+    'pseudogene_db_port'   => $self->o('databases_port'),
+
+    'ncrna_db_server' => $self->o('databases_server'),
+    'ncrna_db_port'   => $self->o('databases_port'),
+    ncrna_db_name     => $self->o('dbowner') . '_' . $self->o('production_name') . '_ncrna_' . $self->o('release_number'),
+
+    'final_geneset_db_server' => $self->o('databases_server'),
+    'final_geneset_db_port'   => $self->o('databases_port'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release'      => $ENV{ENSEMBL_RELEASE},     # this is the current release version on staging to be able to get the correct database
+    'production_db_server' => 'mysql-ens-meta-prod-1',
+    'production_db_port'   => '4483',
+
+    databases_to_delete => [ 'cdna_db', 'genblast_db', 'selected_projection_db', 'layering_db', 'utr_db', 'genebuilder_db', 'pseudogene_db', 'ncrna_db', 'final_geneset_db' ],    #, 'projection_realign_db'
+
+######################################################
+#
+# Mostly constant settings
+#
+######################################################
+
+    genome_dumps => catdir( $self->o('output_path'), 'genome_dumps' ),
+    # This one is used in replacement of the dna table in the core db, so where analyses override slice->seq. Has simple headers with just the seq_region name. Also used by bwa in the RNA-seq analyses. Not masked
+    faidx_genome_file => catfile( $self->o('genome_dumps'), $self->o('species_name') . '_toplevel.fa' ),
+
+    'layering_input_gene_dbs' => [
+      $self->o('selected_projection_db'),
+      $self->o('long_read_final_db'),
+    ],
```

Comment:
I would expect this array to contain more databases than this. Could you please check again with the original Genome_annotation_conf.pm file?

Reply:
I added layering_input_gene_dbs as in the original config file.
I added connections for genblast_nr_db, genblast_rnaseq_support_nr_db, rnaseq_for_layer_nr_db, ig_tr_db, best_targeted_db and the related parameters.
Could you please check the new lines?

Reply:
Done